site2/docs/client-libraries-go.md

```diff
@@ -302,6 +302,80 @@ fmt.Println(msg.Payload())
 canc()
 ```
 
+#### How to use Prometheus metrics in producer
+
+Pulsar Go client registers client metrics using Prometheus. In this guide, we'll show you how to create a simple Pulsar producer application that exposes Prometheus metrics via HTTP.
+
+First, write a simple producer application:
+
+```go
+// Create a pulsar client
```

Comment:
```suggestion
// Create a Pulsar client
```
Reason for this change: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.1oxiv9hq05mn